### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by GodsBoss**
### Motivation/Abstract

There were some cosmetic issues and also some dead code. Imports weren't formatted with [`goimports`](https://pkg.go.dev/golang.org/x/tools/cmd/goimports), for example.

### Description/Comments

Imports are now formatted with `goimports`, did a few other changes that keep the behaviour of the code. I did not create an issue because that seemed unnecessarely bureaucratic to me.

### Checklist

Depending on your PR, please ensure the overall consistency/integrity of the project remains.
Please tick just one check item per section below

#### Tests
- [ ] did you update or create tests for your code changes?
- [X] not relevant

#### Code examples
- [ ] did you update or add example code snippets, which relate to your code changes
- [X] not relevant

#### Documentation
- [ ] did you update or create documentation, which relates to your code changes
- [X] not relevant

### CODE REVIEW

File: pkg/bpmn_engine/timer.go [hunk 1]
```diff
@@ -124,10 +125,5 @@ func findExistingTimerNotYetTriggered(state *BpmnEngineState, id string, instanc
 }
 
 func findDurationValue(ice BPMN20.TIntermediateCatchEvent) (duration.Duration, error) {
-	durationStr := &ice.TimerEventDefinition.TimeDuration.XMLText
-	if durationStr == nil {
-		return duration.Duration{}, newEngineErrorf("Can't find 'timeDuration' value for INTERMEDIATE_CATCH_EVENT with id=%s", ice.Id)
```
Comment (jinjaghost) [inline pkg/bpmn_engine/timer.go:129]: error message is lost

Comment (GodsBoss) [reply pkg/bpmn_engine/timer.go:129]: `durationStr` could never be `nil`, so that error never actually occured.

**Comment by jinjaghost** [comment]
As spotted by [GodsBoss](https://github.com/GodsBoss), there is no message error issue.

File: pkg/bpmn_engine/timer.go (hunk 1 above)
Comment (nitram509) [reply pkg/bpmn_engine/timer.go:129]: @GodsBoss good catch, if `durationStr` can't be `nil` - but then we need to rewrite the if clause, to capture the case when the text is empty. The goal here is to capture required and missing input as early as possible and provide good error message to the user.
If you agree, would you please bring the check back?

**Review by nitram509 — CHANGES_REQUESTED** [review]

File: pkg/bpmn_engine/timer.go (hunk 1 above)
Comment (GodsBoss) [reply pkg/bpmn_engine/timer.go:129]: @nitram509 I'd rather have this PR only change cosmetics, not behaviour. If you wish I can remove that particular change.

Comment (nitram509) [reply pkg/bpmn_engine/timer.go:129]: My point was rather to rework this change towards a fail fast check and not removing "dead code".
Don't worry, I will merge it and then bring back the change on my own ... that round trip is shorter.

**Comment by nitram509** [comment]
@GodsBoss thank you for your contribution 🚀

**Pull request merged**